components/dashboard/src/settings/Teams.tsx

```diff
@@ -527,6 +528,16 @@ function AllTeams() {
                 <div className="flex-grow ">
                     <h3 className="self-center">All Teams</h3>
                     <h2>Manage teams and team members.</h2>
+                    <ItemsList className="pt-2">
+                        <Item className="w-full bg-yellow-200 hover:bg-yellow-200 py-6 px-6">
+                            <ItemField className=" flex flex-col">
+                                <div className="text-yellow-600 font-semibold">DEPRECATION NOTICE</div>
+                                <p className="text-yellow-800">
+                                    Hey! We're introducing <strong>Teams & Projects</strong> to surface <strong>Prebuilds</strong> in the dashboard. All existing teams will be migrated over the upcoming weeks. There's no action required on your side. <a href="/new" className="learn-more">Learn more about Teams & Projects</a> or <a href="/new" className="learn-more">Create a New Team</a>.
```

Comment:
**suggestion**: Minor rephrasing to match changes in https://github.com/gitpod-io/gitpod/pull/5126/files#diff-45333399769993ee06ff1de23fb9aaea15cc06ec5a941eb8fef46421082a589eR18:

```suggestion
                                    Hey! We're introducing <strong>Teams & Projects</strong> to surface <strong>Prebuilds</strong> in the dashboard. All existing teams will be migrated over the upcoming weeks. No action is required on your side. <a href="/new" className="learn-more">Learn more about Teams & Projects</a> or <a href="/new" className="learn-more">Create a New Team</a>.
```